docs/src/main/sphinx/connector/hive.rst

```diff
@@ -1370,6 +1370,14 @@ functionality:
 * Support all Hive data types and correct mapping to Trino types
 * Ability to process custom UDFs
 
+.. _hive-fte-support:
+
+Fault-tolerant execution support
+--------------------------------
+
+The connector supports :doc:`/admin/fault-tolerant-execution` of queries to
+Hive. Read and write operations are both supported with any retry policy.
```

Comment:
No .. thats already covered in other sections .. if necessary this is out of scope and belongs in a different PR

Reply:
This is **not** covered in any other sections of the Hive connector docs.

Reply:
ok .. can you file an issue.. we need to cover this in a separate PR for the Hive connector docs.

Reply:
I agree with @beapirate that this should be mentioned here, because it means enabling FTE on a Hive catalog will effectively disable write and CTAS operations on transactional Hive tables. This is the most appropriate place in the documentation to describe that behavior, and since this PR is intended to describe per-connector support I'm comfortable including it in scope for the commit.